ui/app/AppLayouts/Communities/popups/FinaliseOwnershipPopup.qml

```diff
@@ -0,0 +1,335 @@
+
+                            text: qsTr("Visit Community")
+                            font.pixelSize: Style.current.additionalTextSize
+                            color: Theme.palette.primaryColor1
+                        }
+                    }
+                ]
+
+                onClicked: {
+                    close()
+                    root.visitCommunityClicked()
+                }
+            }
+        }
+    }
+
+    Component {
+        id: finalisePanel
+
+        ColumnLayout {
+            spacing: Style.current.padding
+
+            CustomText {
+                text: qsTr("Finalising your ownership of the %1 Community requires you to:").arg(root.communityName)
+            }
+
+            CustomText {
+                text: qsTr("1. Make this device the control node for the Community")
+                font.bold: true
+            }
+
+            CustomText {
+                Layout.topMargin: -Style.current.halfPadding
+
+                text: qsTr("It is vital to keep your device online and running Status in order for the Community to operate effectively. Login with this account via another synced desktop device if you think it might be better suited for this purpose.")
+            }
+
+            CustomText {
+                text: qsTr("2. Update the %1 Community smart contract").arg(root.communityName)
+                font.bold: true
+            }
+
+            CustomText {
+                Layout.topMargin: -Style.current.halfPadding
+
+                text: qsTr("This transaction updates the %1 Community smart contract, making you the %1 Community owner.").arg(root.communityName)
+            }
+
+            FeesBox {
+                Layout.fillWidth: true
+
+                implicitWidth: 0
+
+                accountsSelector.model: root.accounts
+                accountErrorText: root.feeErrorText
+                accountSelectorText: qsTr("Gas fees will be paid from")
+
+                model: QtObject {
+                    id: singleFeeModel
+
+                    readonly property string title: root.feeLabel
+                    readonly property string feeText: root.isFeeLoading ? "" : root.feeText
+                    readonly property bool error: root.feeErrorText !== ""
+                }
+
+                accountsSelector.onCurrentIndexChanged: {
+                    if (accountsSelector.currentIndex < 0)
+                        return
+
+                    const item = ModelUtils.get(accountsSelector.model,
+                                                accountsSelector.currentIndex)
+                    d.accountAddress = item.address
+                    d.accountName = item.name
+                }
+            }
+
+            StatusCheckBox {
+                Layout.topMargin: -Style.current.halfPadding
+                Layout.fillWidth: true
+
+                checked: d.ackCheck
+                font.pixelSize: Style.current.primaryTextFontSize
```

Comment:
Hehe, I fixed that in https://github.com/status-im/status-desktop/pull/12252/commits/ad4f20578950c4cda5c566311f37fbf3bcc9391d

Needs review :)